clusterctl/CONTRIBUTING.md

```diff
@@ -1,26 +1,15 @@
 # Contributing Guidelines
 
-1. Follow the [Getting Started]((https://github.com/kubernetes-sigs/cluster-api/blob/master/cluster-api/clusterctl/README.md)) steps to create a cluster.
-
-# Development
-
-Before submitting an PR you should run the unit and integration tests. Instructions for doing so are given in the [Testing](#Testing) section.
+Before submitting an PR you should run the unit and integration tests.
 
 ## Testing
 
 ### Unit Tests
-When changing this application, you will often end up modifying other packages above this folder in the project tree. You
-should run all the unit tests in the repository. To run the unit tests, run the following command from the root,
-`cluster-api`, folder of the repo.
 
-```
-go test ./...
-```
-
-### Integration Tests
+When changing this application, you will often end up modifying other packages above this folder in the project tree.
+You should run all the tests in the repository. To run the tests, run the following command from the root,
+`cluster-api`, folder of the repo.
 
-To run the integration tests, run the following command from this folder. The integration tests are for sanity checking
-that clusterctl's basic functionality is working.
+```shell
+./scripts/ci-test.sh
```

Comment:
should we doc the other 2 scripts? ci-build.sh	ci-make.sh

Reply:
sure, I've added them.